ballerina/http_response.bal

```diff
@@ -468,14 +475,26 @@ public class Response {
         self.setEntityAndUpdateContentTypeHeader(entity);
     }
 
+    # Sets an `http:SseEvent` stream as the payload, along with the Content-Type and Cache-Control 
+    # headers set to 'text/event-stream' and 'no-cache', respectively.
+    #
+    # + eventStream - SseEvent stream, which needs to be set to the response
+    public isolated function setSseEventStream(stream<SseEvent, error?>|stream<SseEvent, error> eventStream) {
+        ResponseCacheControl cacheControl = new;
+        cacheControl.noCache = true;
+        self.cacheControl = cacheControl;
```

Comment:
It is possible to overwrite the content-type or cache control after setting the event stream. In those cases, how are we handling them?

Reply:
Yes, it's possible if users need to do that, they can directly manipulate the response object and directly call the caller respond on it.